packages/nuxt/src/app/nuxt.ts

```diff
@@ -119,6 +120,7 @@ export function createNuxtApp (options: CreateOptions) {
   const nuxtApp: NuxtApp = {
     provide: undefined,
     globalName: 'nuxt',
+    version: __NUXT_VERSION__,
```

Comment:
Maybe we could expose it as `versions: { nuxt, vue }`

Reply:
Sounds nice - and maybe also allow other modules/libraries to expose their versions too